pkg/cli/images.go

```diff
@@ -127,52 +137,57 @@ func (a *Image) Run(cmd *cobra.Command, args []string) error {
 }
 
 type imagePrint struct {
+	Project    string `json:"project,omitempty"`
 	Name       string `json:"name,omitempty"`
 	Digest     string `json:"digest,omitempty"`
 	Repository string `json:"repository,omitempty"`
 	Tag        string `json:"tag,omitempty"`
 }
 
 type imageContainer struct {
-	Container string
-	Repo      string
-	Tags      []string
-	Digest    string
-	ImageID   string
+	Container  string
+	Repository string
```

Comment:
Why does this need to change?

Reply:
Naming consistency with imagePrint and imageContainerPrint. Could change it back if desired.